src/main/webapp/WEB-INF/tags/proposaldevelopment/proposalDevelopmentAbstracts.tag

```diff
@@ -24,7 +24,18 @@
 <kul:tab tabTitle="Abstracts" defaultOpen="false" 
          tabItemCount="${fn:length(KualiForm.document.developmentProposalList[0].proposalAbstracts)}" 
          tabErrorKey="document.developmentProposalList[0].proposalAbstract*,newProposalAbstract*">
-         
+	<script>
+		// Add event to replace unicode characters.
+		jQuery(document).ready(function(){
+			// Get abstract details textarea object.
+			var txt = document.getElementById("abstracts-table").getElementsByTagName("textarea")[0];
+
+			// Add keyup event.
+			txt.onkeyup = function(){
+				txt.value = txt.value.replace(/[^A-Za-z 0-9 \.,\?""!@#\$%\^&\*\(\)-_=\+;:<>\/\\\|\}\{\[\]`~]*/g, '')
+			};
+		});
+	</script>
```

Comment:
Alex, please create a .js function in /kuali-coeus-5.2.1/src/main/webapp/scripts/ua_kuali_application.js that contains the char replacement above  (see var replaceWordChars = function(taElement)  for example from the same file) and remove the <script> element from the proposalDevelopmentAbstracts.tag file
Next step: bring in /5.2.1-kc_project/src/main/webapp/WEB-INF/tags/kr/htmlControlAttribute.tag from the foundation project inside ua custom and make the call to your function in this .tag file.as specified in Jennifer's notes.